data/json/bionics.json

```diff
@@ -1017,8 +1013,8 @@
     "id": "bio_surgical_razor",
     "type": "bionic",
     "name": { "str": "Autonomous Surgical Scalpels" },
-    "description": "A system of surgical grade scalpels is implanted on your fingers.  They allow you to make automated precise cuts and can also be used as a high-quality butchering tool.",
-    "occupied_bodyparts": [ [ "arm_r", 1 ], [ "arm_l", 1 ] ],
+    "description": "A system of surgical grade scalpels is implanted in your wrists.  They allow you to make automated precise cuts and can also be used as a high-quality butchering tool.",
```

Comment:
Manipulating a scapel from your wrist would be incredibly akward

Reply:
I can either edit the description to something like "wrist-mounted scalpels with finger manipulators" or rollback that one.
![image](https://user-images.githubusercontent.com/26396019/102783430-34d07280-439b-11eb-9f64-1a9728831014.png)

Reply:
if they were wrist-mounted with finger manipulators, they would extend across the whole palm, effectively crippling it. I suggest rolling back.

Reply:
These are specified to be autonomous, so they really only need to be someplace that the person can easily move around; I suggest having them extend out from (slightly above) the wrist as tentacles stored in the forearm (between the ulna and radius).

Reply:
They were already occupying arm slots, so I could switch it to use hand slots and rollback increase in occupied slots.

I don't get how installing small blades on your fingertips would be so good with butchering and cutting.
"They allow you to make automated precise cuts" - so are they automated device?
Now I'm thinking something like 
![image](https://user-images.githubusercontent.com/26396019/102810606-509c3e80-43c4-11eb-9c44-fd03ee636643.png)

Reply:
Current version is:
```
"description": "A system of surgical grade scalpels is implanted in your forearms, extending from just above your wrists.....
"occupied_bodyparts": [ [ "arm_r", 2 ], [ "arm_l", 2 ] ],
```
So compared to what's currently in the game I changed the description and increased the slots used.